docs/faq.rst

```diff
@@ -261,7 +261,10 @@ How do I add other users as maintainers of a tool?
 For tools registered on Quay.io and workflows registered with GitHub,
 Dockstore automatically allows users from the same Quay.io organization
 or GitHub organization to manage tools/workflows together (users will
-need to "Refresh Organization" or "Refresh All").
+need to click "Discover Existing Dockstore Workflows" in the "My workflows"
```

Comment:
I know the JIRA ticket suggested this language, but it doesn't read very well to me, mainly because of the way it already was.

1. The FAQ item is titled "How do I add other users as maintainers of a tool?" But then the answer discusses both tools and workflows. I suggest either changing the question to "How do I add other users as maintainers of a tool or worklow?", or creating a new item "How do I add other users as maintainers of a workflow?".
2. If we keep them under the same item, I suggest having separate paragraphs for tools and workflows.
3. I think the case this is covering is somebody else in your GitHub org registers a workflow with GitHub apps, but it doesn't show up under your My Workflows. I feel it could be clearer.

Reply:
> "How do I add other users as maintainers of a tool or worklow?"

Changing the question seems good to me

> I think the case this is covering is somebody else in your GitHub org registers a workflow with GitHub apps, but it doesn't show up under your My Workflows. I feel it could be clearer. 

I believe it is

Reply:
updated question in 708e8fa

Reply:
Re-reading, I guess my comment was ambiguous. I feel that @coverbeck identified the case correctly that this is covering. I wasn't commenting on whether this could be clearer or not. (It probably could)

Reply:
I think this answer is pretty confusing. I'm not sure why it mentions tools, but doesn't give an answer for tools. I don't think there's even an equivalent button to "Discover Existing Dockstore Workflows" for tools?